packages/devtools_app/lib/src/screens/inspector/layout_explorer/flex/flex.dart

```diff
@@ -661,6 +661,10 @@ class FlexChildVisualizer extends StatelessWidget {
   }
 
   Widget _buildContent(ColorScheme colorScheme) {
+    final currentFlexFactor = properties.flexFactor?.toInt() ?? 0;
```

Comment:
can you link a TODO to the follow up issue you created

Reply:
ah yes, I've got to get used to doing this!